pageserver/src/lib.rs

```diff
@@ -81,7 +79,7 @@ pub async fn shutdown_pageserver(exit_code: i32) {
 
     // There should be nothing left, but let's be sure
     task_mgr::shutdown_tasks(None, None, None).await;
-
+    std::thread::sleep(Duration::from_secs(4));
```

Comment:
I have probably missed some earlier discussion on why was this added, but adding a comment just in case.

Reply:
Or, now seeing comments about tests getting longer, maybe this is the reason? If this needs to be kept, would prefer `tokio::time::sleep(...).await;` of course :)

Reply:
Alas, no `tokio` here around and that should be a separate PR.

Reply:
I don't understand why no tokio? This is an async function, and we have just awaited on above line.

Could a comment be added on why the sleep is needed?

Reply:
Apologies, have misunderstood your original comment.
The line is gone now, it does not belong here.